data-access-layer/bosh/BoshDirectorClient.js

```diff
@@ -829,12 +829,7 @@ class BoshDirectorClient extends HttpClient {
   getTask(taskId) {
     const splitArray = this.parseTaskid(taskId);
     if (splitArray === null) {
-      return this
-        .makeRequestWithConfig({
-          method: 'GET',
-          url: `/tasks/${taskId}`
-        }, 200, this.getConfigByName(CONST.BOSH_DIRECTORS.BOSH))
-        .then(res => JSON.parse(res.body));
+      throw new errors.UnprocessableEntity(`not able to query correct bosh as taskId is not in required format: ${taskId}`);
```

Comment:
Just start the statement in upper case character and add UT. Rest looks fine.

Reply:
Will do. Thanks